[core]/es_extended/client/functions.lua

```diff
@@ -507,10 +507,10 @@ function ESX.Game.GetClosestEntity(entities, isPlayerEntities, coords, modelFilt
 
     if modelFilter then
         filteredEntities = {}
-
-        for _, entity in pairs(entities) do
-            if modelFilter[GetEntityModel(entity)] then
-                filteredEntities[#filteredEntities + 1] = entity
+    
+        for current_entity_index = 1, #entities do
```

Comment:
Please use snake_case instead of lowerCamelCase name convection.

Reply:
You mean lowerCamelCase instead of snake case right?

Reply:
yes, exactly we don't use snake case in es_extended

Reply:
i will change it, probably tomorrow